cmd/cli.go

```diff
@@ -366,6 +278,63 @@ var containerdDumpCmd = &cobra.Command{
 	},
 }
 
+var gpuDumpCmd = &cobra.Command{
```

Comment:
can you move these to the process checkpoint/restore comment block? And then also add a note about how this is for processes only right now.

Reply:
Done!